pkg/helm/chart.go

```diff
@@ -161,6 +162,7 @@ func (c *ActionableChart) Upgrade(chart *chart.Chart, values map[string]interfac
 
 func (c *ActionableChart) Uninstall() (*release.UninstallReleaseResponse, error) {
 	client := action.NewUninstall(c.config)
+	client.Wait = true
```

Comment:
We should probably specify a longer timeout to this, I don't know if the default is long enough for everything.